### ISSUE TITLE
Make it possible to navigate popover menus with arrow keys

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#753]
When opening a popover menu such as the user menu or the "sort" menu on series blocks, users should be able to navigate items with the up/down arrow keys, not just the tab key.
